build.js

```diff
@@ -2,11 +2,11 @@ const StyleDictionary = require('style-dictionary')
   .extend(`${__dirname}/config.js`);
 const fs = require('fs');
 const _ = require('lodash');
-const tokens = require('./designTokens/spacings');
-const baseSpacing = tokens.base.spacing.value;
+const tokens = require('./designTokens/spacing');
```

Comment:
a bit misleading, i would rather name it `spacings` or `spacingTokens` or `tokensSpacing`